pkg/wikidata/identifier.go

```diff
@@ -155,6 +156,7 @@ func (i *Identifier) Fields() []string {
 		"id",
 		"format",
 		"URI",
+		"wikibase",
```

Comment:
Will become `Permalink` however, this structmap will be removed with the next release in favor of the one below. See https://github.com/richardlehane/siegfried/pull/162